### ISSUE TITLE
How to get the message ID

### ISSUE THREAD

**Issue by gujingc** [buddypress/BP-REST#385]
We are building a mobile app, in which forum is a key function.

When the forum a user subscribes to has new reply or new topic, the user will receive a notification.

We would like the user click on the notification and be redirected to the forum new forum.

To achieve this on the mobile app, we need message ID to redirect the user to the right screen. 

We are using BP Rest API. How to obtain the message ID? Please advise.

**Comment by renatonascalves** [comment]
I don't follow! Are you talking about the Messages component in BuddyPress? Creating a private message between two users? If so, here is the relevant doc https://developer.buddypress.org/bp-rest-api/reference/private-messaging/#retrieve-a-specific-messages-thread to retrieve a specific message from a thread.

But based on your question, it seems you are doing something with bbPress and not BuddyPress. If that's the case, the question is unrelated.

I'm closing as answered or invalid. Feel free to reply back with more information. :)

**Comment by gujingc** [comment]
Thank you for writing back! 

We are building an app which is powered by our site which has both bbPress and BuddyPress. When a user posts on the forum, the user will receive a notification. 

As the user clicks on the notification, the app should redirect the user to the screen where the new post is posted on the forum. 

And we ran into a problem that we could not get the screen ID. it appears to us to achieve this function, we would need the bp-REST-API to be extended. 

We found this document which indicates the bp-REST-API can be easily extended: [https://developer.buddypress.org/bp-rest-api/extending-the-bp-rest-api/](url)
Is this correct?
